components/lib/api/PrimeIcons.js

```diff
@@ -1,263 +1,316 @@
-    SORT_AMOUNT_UP: 'pi pi-sort-amount-up',
-    SORT_AMOUNT_UP_ALT: 'pi pi-sort-amount-up-alt',
-    SORT_DOWN: 'pi pi-sort-down',
-    SORT_NUMERIC_DOWN: 'pi pi-sort-numeric-down',
-    SORT_NUMERIC_DOWN_ALT: 'pi pi-sort-numeric-down-alt',
-    SORT_NUMERIC_UP: 'pi pi-sort-numeric-up',
-    SORT_NUMERIC_UP_ALT: 'pi pi-sort-numeric-up-alt',
-    SORT_UP: 'pi pi-sort-up',
-    SPINNER: 'pi pi-spinner',
-    STAR: 'pi pi-star',
-    STAR_FILL: 'pi pi-star-fill',
-    STEP_BACKWARD: 'pi pi-step-backward',
-    STEP_BACKWARD_ALT: 'pi pi-step-backward-alt',
-    STEP_FORWARD: 'pi pi-step-forward',
-    STEP_FORWARD_ALT: 'pi pi-step-forward-alt',
-    STOP: 'pi pi-stop',
-    STOP_CIRCLE: 'pi pi-stop-circle',
-    STOPWATCH: 'pi pi-stopwatch',
-    SUN: 'pi pi-sun',
-    SYNC: 'pi pi-sync',
-    TABLE: 'pi pi-table',
-    TABLET: 'pi pi-tablet',
-    TAG: 'pi pi-tag',
-    TAGS: 'pi pi-tags',
-    TELEGRAM: 'pi pi-telegram',
-    TH_LARGE: 'pi pi-th-large',
-    THUMBS_DOWN: 'pi pi-thumbs-down',
-    THUMBS_DOWN_FILL: 'pi pi-thumbs-down-fill',
-    THUMBS_UP: 'pi pi-thumbs-up',
-    THUMBS_UP_FILL: 'pi pi-thumbs-up-fill',
-    TICKET: 'pi pi-ticket',
-    TIMES: 'pi pi-times',
-    TIMES_CIRCLE: 'pi pi-times-circle',
-    TRASH: 'pi pi-trash',
-    TRUCK: 'pi pi-truck',
-    TWITTER: 'pi pi-twitter',
-    UNDO: 'pi pi-undo',
-    UNLOCK: 'pi pi-unlock',
-    UPLOAD: 'pi pi-upload',
-    USER: 'pi pi-user',
-    USER_EDIT: 'pi pi-user-edit',
-    USER_MINUS: 'pi pi-user-minus',
-    USER_PLUS: 'pi pi-user-plus',
-    USERS: 'pi pi-users',
-    VERIFIED: 'pi pi-verified',
-    VIDEO: 'pi pi-video',
-    VIMEO: 'pi pi-vimeo',
-    VOLUME_DOWN: 'pi pi-volume-down',
-    VOLUME_OFF: 'pi pi-volume-off',
-    VOLUME_UP: 'pi pi-volume-up',
-    WALLET: 'pi pi-wallet',
-    WHATSAPP: 'pi pi-whatsapp',
-    WIFI: 'pi pi-wifi',
-    WINDOW_MAXIMIZE: 'pi pi-window-maximize',
-    WINDOW_MINIMIZE: 'pi pi-window-minimize',
-    WRENCH: 'pi pi-wrench',
-    YOUTUBE: 'pi pi-youtube'
+    ADDRESS_BOOK: 'pi .pi-address-book',
+    ALIGN_CENTER: 'pi .pi-align-center',
+    ALIGN_JUSTIFY: 'pi .pi-align-justify',
+    ALIGN_LEFT: 'pi .pi-align-left',
+    ALIGN_RIGHT: 'pi .pi-align-right',
+    AMAZON: 'pi .pi-amazon',
+    ANDROID: 'pi .pi-android',
+    ANGLE_DOUBLE_DOWN: 'pi .pi-angle-double-down',
+    ANGLE_DOUBLE_LEFT: 'pi .pi-angle-double-left',
+    ANGLE_DOUBLE_RIGHT: 'pi .pi-angle-double-right',
+    ANGLE_DOUBLE_UP: 'pi .pi-angle-double-up',
+    ANGLE_DOWN: 'pi .pi-angle-down',
+    ANGLE_LEFT: 'pi .pi-angle-left',
+    ANGLE_RIGHT: 'pi .pi-angle-right',
+    ANGLE_UP: 'pi .pi-angle-up',
+    APPLE: 'pi .pi-apple',
+    ARROW_CIRCLE_DOWN: 'pi .pi-arrow-circle-down',
+    ARROW_CIRCLE_LEFT: 'pi .pi-arrow-circle-left',
+    ARROW_CIRCLE_RIGHT: 'pi .pi-arrow-circle-right',
+    ARROW_CIRCLE_UP: 'pi .pi-arrow-circle-up',
+    ARROW_DOWN_LEFTANDARROW_UP_RIGHT_TO_CENTER: 'pi .pi-arrow-down-left-and-arrow-up-right-to-center',
```

Comment:
I think there are some missing underscores around "AND" here.

Also, I count 311 icons in the documentation – how come 314 entries here?

Reply:
@melloware It is "pi-circle-off", "pi-circle-on" and "pi-file-o" that are defined here but aren't listed in the documentation.

Reply:
I copied the names of the icons right from primeicons.css using regex and those values don't have underscores in the icon name so I kept them exact.

Reply:
"ARROW_UP_RIGHT_AND_ARROW_DOWN_LEFT_FROM_CENTER" does have underscores, though, and reads much nicer.